utils/windowfunction/idborderby.cpp

```diff
@@ -6,8 +6,8 @@
    as published by the Free Software Foundation; version 2 of
    the License.
 
-   This program is distributed in the hope that it will be useful,
-   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   his program is distributed in the hope that it will be useful,
+   ut WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
```

Comment:
Please fix the license edit

Reply:
Sure thing but I will wait for Patrick's review first.

Reply:
Fixed